test/test_ABA_matrix.jl

```diff
@@ -17,7 +17,7 @@
         # evaluate if the ABA matrix is correct
         A = IncidenceMatrix(sys)
         BA = BA_Matrix(sys)
-        ABA_2 = transpose(A.data) * BA.data
+        ABA_2 = transpose(A.data) * BA.data'
```

Comment:
why transpose twice?

Reply:
because ABA = A'*BA. Since the new new BA is evaluated as a transpose matrix, then ABA = A' * BA'